flask_annex/file.py

```diff
@@ -81,3 +81,6 @@ def send_file(self, key):
             as_attachment=True,
             attachment_filename=os.path.basename(key),
         )
+
+    def get_upload_info(self, key):
+        return None
```

Comment:
I had an earlier version of this that took a view name and did route reversing here, but it seem nicer to push that into the consumer.